site/docs/master/style-guide.md

```diff
@@ -0,0 +1,317 @@
+<tr><td>Use meaningful variable names that have a context.</td><td>Use variable names such as 'foo','bar', and 'baz' that are not meaningful and lack context.</td></tr>
+<tr><td>Remove trailing spaces in the code.</td><td>Add trailing spaces in the code, where these are important, because the screen reader will read out the spaces as well.</td></tr>
+</table>
+
+### Starting a sentence with a component tool or component name
+
+<table caption="Do and Don't - Starting a sentence with a component tool or component name" >
+<tr><th>Do</th><th>Don't</th></tr>
+<tr><td markdown="span">The `kubeadm` tool bootstraps and provisions machines in a cluster.</td><td markdown="span">`kubeadm` tool bootstraps and provisions machines in a cluster.</td></tr>
+<tr><td>The kube-scheduler is the default scheduler for Kubernetes.</td><td>kube-scheduler is the default scheduler for Kubernetes.</td></tr>
+</table>
+
+### Use normal style for string and integer field values
+
+For field values of type string or integer, use normal style without quotation marks.
+
+<table caption="Do and Don't - Use normal style for string and integer field values" >
+<tr><th>Do</th><th>Don't</th></tr>
+<tr><td markdown="span">Set the value of `imagePullPolicy` to `Always`.</td><td markdown="span">Set the value of `imagePullPolicy` to "Always".</td></tr>
+<tr><td markdown="span">Set the value of `image` to `nginx:1.16`.</td><td markdown="span">Set the value of `image` to nginx:1.16.</td></tr>
+<tr><td markdown="span">Set the value of the `replicas` field to `2`.</td><td markdown="span">Set the value of the `replicas` field to 2.</td></tr>
+</table>
+
+## Code snippet formatting
+
+
+### Don't include the command prompt
+
+<table caption="Do and Don't - Don't include the command prompt" >
+<tr><th>Do</th><th>Don't</th></tr>
+<tr><td>kubectl get pods</td><td>$ kubectl get pods</td></tr>
+</table>
+
+### Separate commands from output
+
+Verify that the Pod is running on your chosen node:
+
+```
+kubectl get pods --output=wide
+```
+
+The output is similar to this:
+
+```
+NAME     READY     STATUS    RESTARTS   AGE    IP           NODE
+nginx    1/1       Running   0          13s    10.200.0.4   worker0
+```
+
+## Velero.io word list
+
+
+A list of Velero-specific terms and words to be used consistently across the site.
+
+<table caption="Velero.io word list" >
+<tr><th>Trem</th><th>Useage</th></tr>
+<tr><td>Kubernetes</td><td>Kubernetes should always be capitalized.</td></tr>
+<tr><td>Docker</td><td>Docker should always be capitalized.</td></tr>
+<tr><td>Velero</td><td>Velero should always be capitalized.</td></tr>
+<tr><td>VMware</td><td>VMware should always be correctly capitalized.</td></tr>
+<tr><td>On-premises</td><td>On-premises or on-prem rather than on-premise or other variations.</td></tr>
+<tr><td>Backup</td><td>Backup rather than back up, back-up or other variations.</td></tr>
```

Comment:
We've been using `backup` as the artifact produced by the process, and `back up` as the verb for the process itself.

I'm happy to converge on the same style for both, though, as having the separately formatted words makes it much harder for folks to know which is correct for the context.

Reply:
I think its better to have a single spelling